content/java/concepts/hash-map/hash-map.md

```diff
@@ -0,0 +1,217 @@
+---
+Title: 'HashMap'
+Description: 'In Java, a HashMap is used to store items as a "key/value" pairs. The keys and values can be of either same or different types. java // Create a HashMap which stores courses and their teachers as a key/value pair  HashMap<String, String> courseTeacher = new HashMap<String, String>();'
```

Comment:
```suggestion
Description: 'Used to store items as a key-value pairs. The keys and values can be of either same or different types.'
```

A few quick notes:

- Per [our content standards regarding meta-data](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), the 'Description' should be (ideally) 150 characters or less. 
- The `Description` doesn't have to mirror the opening sentences verbatim. 
- Lastly, it should begin with a verb in the present-tense. (~~In Java, a HashMap is used to...~~ -> Used to...)